src/index.js

```diff
@@ -1,133 +1,95 @@
-window.Realm = class {
-    constructor() {
-        const iframe = this.#iframe;
-        iframe.setAttribute('sandbox', 'allow-same-origin allow-scripts');
-        iframe.style.display = 'none';
-
-        document.body.appendChild(iframe);
-
-        const { contentWindow } = iframe;
-        this.#RedFunction = contentWindow.Function;
-        this.#RedEval = contentWindow.eval;
-
-        iframe.remove();
-
-        this.#fakeIntrinsic;
-    }
-
-    #RedEval;
-    #RedFunction;
-
-    // #BlueFunction = Function;
-    // #BlueAsyncFunction = (async function () {}).constructor;
-    // #BluePromise = Promise;
-
-    #iframe = document.createElement('iframe');
-
-    // This simulates the `%Realm%` preserved value
-    #fakeIntrinsic = this.constructor;
-
-    #isPrimitive(value) {
-        return value === null || (typeof value !== 'function' && typeof value !== 'object');
-    }
-
-    eval(str) {
-        var res = this.#errorCatcher(() => this.#RedEval(str));
+{
+
+    // const wm = new WeakMap();
```

Comment:
Whoops, let's get rid of that. Sorry for leaving my trash everywhere

Reply:
this got me puzzled for so long trying to think what I should be using this for. I thought of adding wrapped functions there, but maybe I'm missing something I still ignore.